<ui version="4.0" >
 <class>MediaflyMenu</class>
 <widget class="QWidget" name="MediaflyMenu" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="m_header" >
     <property name="font" >
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="m_icon" >
     <property name="minimumSize" >
      <size>
       <width>101</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>101</width>
       <height>111</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../resources/mediafly.qrc" >:/logo.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="mf::List" name="m_listView" >
     <property name="font" >
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mf::List</class>
   <extends>QListView</extends>
   <header>List.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/mediafly.qrc" />
 </resources>
 <connections/>
</ui>
